qemu/tests/blockdev_commit_speed_limit.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import random
+
+from virttest.qemu_monitor import QMPCmdError
+
+from provider import backup_utils
+from provider import job_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitSpeedLimit(BlockDevCommitTest):
+
+    def commit_snapshots(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["speed"]
+        arguments = self.params.copy_from_keys(options)
+        arguments["speed"] = self.params["speed"]
+        device = self.get_node_name(snapshot_tags[-1])
+        commit_cmd = backup_utils.block_commit_qmp_cmd
+        cmd, args = commit_cmd(device, **arguments)
+        self.main_vm.monitor.cmd(cmd, args)
+        job_id = args.get("job-id", device)
+        job_utils.check_block_jobs_started(self.main_vm, [job_id])
+        commit_speed = self.params.get("commit_speed",
+                                       str(random.randint(1, 2147483647)))
```

Comment:
cfg:
small_speed = 1
large_speed = 2147483647
commit_speed = ${small_speed} for small_int variant, while
commit_speed = ${large_speed} for large_int variant
here:
random.randint(self.params.get_numeric('small_speed'), self.params.get_numberic('large_speed'))

Reply:
Okay,thanks.